recipe/meta.yaml

```diff
@@ -22,7 +22,7 @@ requirements:
   run:
     - boto3
     - filelock
-    - numpy
+    - numpy <2.0.0
```

Comment:
Not liking this upper pin (because even if we set it here, the conda solver might still pick up litdata<=v0.2.11). Gonna wait for `v0.2.13` or newer and see if they release a patch with Numpy 2.0 compatibility.

Reply:
Opened PR at https://github.com/Lightning-AI/litdata/pull/230 for NumPy 2.0 compatibility